src/annotator/test/integration/anchoring-test.js

```diff
@@ -1,7 +1,7 @@
 // Tests that the expected parts of the page are highlighted when annotations
 // with various combinations of selector are anchored.
 
-import Guest from '../../guest';
+import Guest, { $imports } from '../../guest';
```

Comment:
Since this is not a unit test which corresponds to a single module, where the meaning of `$imports` would be obvious, I think it would be a good idea to give this a more specific name, eg. `guestImports`. Also if you ever need to mock imports from additional modules in this test, you'll have a pattern to follow.